client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -73,377 +73,466 @@ module.exports = protectForm( React.createClass( {
-						autoCapitalize="none"
-						onClick={ this.recordEvent.bind( this, 'Clicked Blacklist Field' ) }
-						onKeyPress={ this.recordEventOnce.bind( this, 'typedBlacklistKeys', 'Typed In Blacklist Field' ) }>
-					</FormTextarea>
-				</p>
+				<FormLegend>{ this.translate( 'Comment Blacklist' ) }</FormLegend>
+				<FormLabel htmlFor="blacklist_keys">{ this.translate(
+					'When a comment contains any of these words in its content, name, URL, e-mail, or IP, it will be marked as spam. ' +
+					'One word or IP per line. It will match inside words, so "press" will match "WordPress".'
+				) }</FormLabel>
+				<FormTextarea
+					name="blacklist_keys"
+					id="blacklist_keys"
+					value={ this.state.blacklist_keys }
+					onChange={ this.handleText }
+					disabled={ this.state.fetchingSettings }
+					autoCapitalize="none"
+					onClick={ this.recordEvent.bind( this, 'Clicked Blacklist Field' ) }
+					onKeyPress={ this.recordEventOnce.bind( this, 'typedBlacklistKeys', 'Typed In Blacklist Field' ) }>
+				</FormTextarea>
 			</FormFieldset>
 		);
 	},
 
-	render: function() {
+	renderInputNumLinks() {
 		return (
+			<FormTextInput
+				name="comment_max_links"
+				type="number"
+				step="1"
+				min="0"
+				className="small-text"
+				value={ 'undefined' === typeof this.state.comment_max_links ? 2 : this.state.comment_max_links }
+				onChange={ this.handleText }
+				disabled={ this.state.fetchingSettings }
+				onClick={ this.recordEvent.bind( this, 'Clicked Comment Queue Link Count Field' ) }
+				onKeyPress={ this.recordEventOnce.bind( this, 'typedCommentQueue', 'Typed In Comment Queue Link Count Field' ) } />
+		);
+	},
 
-			<form id="site-settings" onSubmit={ this.handleSubmitForm } onChange={ this.props.markChanged }>
-				<SectionHeader label={ this.translate( 'Discussion Settings' ) }>
+	renderSectionHeader( title, showButton = true ) {
+		return (
+			<SectionHeader label={ title }>
+				{ showButton &&
 					<Button
-						primary
 						compact
-						disabled={ this.state.fetchingSettings || this.state.submittingForm }
-						onClick={ this.handleSubmitForm }>
+						primary
+						onClick={ this.handleSubmitForm }
+						disabled={ this.state.fetchingSettings || this.state.submittingForm }>
 						{ this.state.submittingForm ? this.translate( 'Saving…' ) : this.translate( 'Save Settings' ) }
 					</Button>
-				</SectionHeader>
-				<Card className="discussion-settings">
+				}
+			</SectionHeader>
+		);
+	},
+
+	render() {
+		return (
+			<form id="site-settings" onSubmit={ this.handleSubmitForm } onChange={ this.props.markChanged }>
+				{ this.renderSectionHeader( this.translate( 'Default Article Settings' ) ) }
+				<Card className="site-settings__discussion-settings">
 					{ this.defaultArticleSettings() }
+				</Card>
+
+				{ this.renderSectionHeader( this.translate( 'Comments' ) ) }
+				<Card className="site-settings__discussion-settings">
 					{ this.otherCommentSettings() }
+					<hr />
```

Comment:
Each section is already wrapped in a `FormFieldset` (see each function), but we want the divider between these sections. I don't want to use the `has-divider` class on `FormFieldset` because I'm also using `FormLegend` (check out `commentModerationSettings()` for an example), and the legend overlaps the border. It seemed more straightforward to switch to `hr`s, rather than trying to jump through CSS hoops or losing semantic HTML.